src/main/drivers/bus_spi.c

```diff
@@ -119,6 +119,9 @@ void initSpi1(void)
     // MISO as input
     gpio.pin = SPI1_MISO_PIN;
     gpio.mode = Mode_IN_FLOATING;
+#ifdef USE_NRF24_SPI1
```

Comment:
Sorry, it's useless, just some debugging artifacts, it can be removed, note that I did the same thing for SPI2 a few lines later.